src/drivers/mpu6000/mpu6000.cpp

```diff
@@ -199,6 +201,13 @@ class MPU6000 : public device::CDev
 	float			_gyro_range_rad_s;
 
 	unsigned		_sample_rate;
+	unsigned		_gyro_control_interval;
+	unsigned		_gyro_count;
+	bool			_gyro_publish_unfiltered;
+	perf_counter_t		_sample_interval_accel;
```

Comment:
I would consider consolidating these are dropping as many as possible. If you profile a nuttx board you'll find they definitely aren't free.